lib/msf/core/db_manager/cred.rb

```diff
@@ -3,13 +3,14 @@ module Msf::DBManager::Cred
   def creds(opts)
     query = nil
     ::ActiveRecord::Base.connection_pool.with_connection {
-      wspace = Msf::Util::DBManager.process_opts_workspace(opts, framework)
-      search_term = opts.delete(:search_term)
       # If :id exists we're looking for a specific record, skip the other stuff
       if opts[:id].present?
```

Comment:
Other modules are performing this check as `if opts[:id] && !opts[:id].empty?` and I'd like to see us move away from Rails helpers.